### PR TITLE
fix(grid): expand/collapse icon is not centered

### PR DESCRIPTION
**Pull request by magdalenaan**
fixes https://github.com/telerik/kendo-theme-bootstrap/issues/82

### CODE REVIEW

**Review by inikolova — COMMENTED** [review]
Can we change this hardcoded value 7px? Maybe we could use some calculations? For example (.k-group-col (which is 30px) - $icon-size (16px)) /2? What do you think?

File: scss/grid/_layout.scss [hunk 1]
```diff
@@ -97,6 +97,11 @@
             text-overflow: none;
         }
 
+        .k-grouping-row .k-icon {
+            margin-left: addTwo(-1*$cell-padding-x, 7px);
```
Comment (joneff) [inline scss/grid/_layout.scss]: change to 7px to `$icon-spacing / 2`

**Review by joneff — CHANGES_REQUESTED** [review]

File: scss/grid/_layout.scss (hunk 1 above)
Comment (magdalenaan) [reply scss/grid/_layout.scss]: I think that there should be used $icon-size variable so icons could be centered in all cases.
If the header [k-group-cell width](https://www.screencast.com/t/TjCqs6jER) is ($icon-size*2), then "7px" can be replaced by ($icon-size/2).

Comment (joneff) [reply scss/grid/_layout.scss]: Agree. You can go ahead with the changes.

**Review by joneff — APPROVED** [review]

**Pull request merged**